config/v1/types_scheduling.go

```diff
@@ -20,7 +19,11 @@ type Scheduling struct {
 }
 
 type SchedulingSpec struct {
-	// default node selector (I would be happy to see this die....)
+	// PolicyConfigMap is a reference to a ConfigMap containing scheduler policy which has
+	// user specified predicates and priorities.
+	// The namespace for this configmap is openshift-config
+	// +optional
+	PolicyConfigMap ConfigMapNameReference `json:"policyConfigMap,omitempty"`
```

Comment:
We avoid hungarian notation.  We know it's a configmap without saying it in the name.

Reply:
Sure @deads2k, I made change. We could have a policy-config-file as well instead of map, so I added it initially. But I deleted it now.